lld/ELF/Relocations.cpp

```diff
@@ -492,7 +492,7 @@ int64_t RelocationScanner::computeMipsAddend(const RelTy &rel, RelExpr expr,
 
   // The ABI says that the paired relocation is used only for REL.
   // See p. 4-17 at ftp://www.linux-mips.org/pub/linux/mips/doc/ABI/mipsabi.pdf
-  if (RelTy::IsRela)
+  if (RelTy::HasAddend)
```

Comment:
Assuming MIPS will support CREL, but the ABI won't be updated to explicitly mention it, could be worth a small addition at the end.
```
The ABI says the paired relocation  is used only for REL.
  // See p. 4-17 at ftp://www.linux-mips.org/pub/linux/mips/doc/ABI/mipsabi.pdf this generalises to relocation types with implicit addends.
```

Reply:
Thx. Added. (I think MIPS will die before anyone implements CREL for it...)